src/emulator/touch/include/touch/touch.h

```diff
@@ -17,10 +17,10 @@
 
 #pragma once
 
-#include <ctrl/state.h>
+#include <host/state.h>
```

Comment:
forward declare, no need for the include.
```suggestion
struct HostState;
```